apps/docs/src/app/docs/[[...slug]]/page.tsx

```diff
@@ -23,7 +23,7 @@ export default async function Page({
   const footer = (
     <>
       <a
-        href={`${gitHubRepoUrl}/blob/main/${path}`}
+        href={`${gitHubRepoUrl}/blob/main/apps/docs/${path}`}
```

Comment:
Nice, great work on tests overall. Mostly clean-up work left to do. I'll run them locally after the review comments are addressed.